packages/radix-ui-themes/src/components/table.css

```diff
@@ -132,3 +153,37 @@
   --scrollarea-scrollbar-horizontal-margin-left: 0;
   --scrollarea-scrollbar-horizontal-margin-right: 0;
 }
+
+/* minimal */
+
+.rt-TableRoot:where(.rt-variant-minimal) {
```

Comment:
I thinking if we can simplify the changes here to achieve the `minimal` visuals, while keeping backwards compatible:

1. Move some of the variables to `.rt-TableRoot` and create a new variable:
```scss
.rt-TableRoot {
  --table-cell-edge-padding: var(--table-cell-padding-inline);
  --table-row-box-shadow: inset 0 -1px var(--gray-a5);
  --table-row-background-color: transparent;
}
```

2. Update the `.rt-TableCell` to apply a specific padding to the left/right side of the first/last child and to use `padding-block/padding-inline` (this is because size-3 uses different padding for block/inline):

```scss
&:where(&:first-child) {
  padding-left: var(--table-cell-edge-padding, var(--table-cell-padding-inline));
}

&:where(&:last-child) {
  padding-left: var(--table-cell-edge-padding, var(--table-cell-padding-inline));
}

/* Inset with Table */
.rt-Inset :where(&:first-child) {
  padding-left: var(--inset-padding-left, var(--table-cell-padding-inline));
}
.rt-Inset :where(&:last-child) {
  padding-right: var(--inset-padding-right, var(--table-cell-padding-inline));
}
```

3. Update the sizes to use the new variables:
```scss
// size 1
--table-cell-padding-inline: var(--space-2);

// size 2
--table-cell-padding-inline: var(--space-3);

// size 3
--table-cell-padding-inline: var(--space-4);
--table-cell-padding-block: var(--space-3);
```

3. Create the minimal variant:

```scss
.rt-TableRoot:where(.rt-variant-minimal) {
  --table-row-box-shadow: none;
  --table-cell-edge-padding: 0px;

  --scrollarea-scrollbar-horizontal-margin-left: 0;
  --scrollarea-scrollbar-horizontal-margin-right: 0;
}
```